packages/components/src/internal/components/permissions/PermissionAssignments.spec.tsx

```diff
@@ -188,7 +191,7 @@ describe('PermissionAssignments', () => {
     test('displays details', async () => {
         const wrapper = mountWithAppServerContext(
             <PermissionAssignments {...getDefaultProps()} />,
-            getDefaultAppContext(),
+            getDefaultAppContext({ fetchPolicy, fetchGroups, getGroupMemberships }),
```

Comment:
Oddly here and on line 239 I have to renew the fns I pass into the defaultAppContext, or the tests start running red.